submissions/AriadnaKravchuk/opp-tiny-js-world/index.js

```diff
@@ -0,0 +1,124 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/AriadnaKravchuk/a-tiny-JS-world
+   Web app: https://ariadnakravchuk.github.io/a-tiny-JS-world
+*/
+
+// ======== OBJECTS DEFINITIONS ========
+
+const SPECIES = {
+   HUMAN: "human",
+   CAT: "cat",
+   DOG: "dog"
+}
+
+const GENDER = {
+   MALE: "male",
+   FEMALE: "female"
+}
+
+const SAYING = {
+   DOG: "woof-woof!",
+   CAT: "meow!"
+}
+
+class Inhabitant {
+   constructor(species, name, gender, saying) {
+      this.species = species;
+      this.name = name;
+      this.gender = gender;
+      this.saying = saying;
+      this.friends = [];
+      this.attributes = [
+         this.species,
+         this.name,
+         this.gender,
+         this.saying
+      ];
```

Comment:
All child classes inherit this property. It is never updated and contains only root class properties. So not much use for it outside the root class.
You can safely form the array of attributes in `Inhabitant#getAttributes` method.